source/prerequisites/postgres.txt

```diff
@@ -1,13 +1,147 @@
 .. _rm-prereq-postgres:
 
-================================
-Postgres Migration Prerequisites
-================================
+==================================
+PostgreSQL Migration Prerequisites
+==================================
 
 .. contents:: On this page
    :local:
    :backlinks: none
    :depth: 1
    :class: singlecol
 
-Postgres.
\ No newline at end of file
+To run sync jobs from an PostGreSQL source database, the database may 
+require some configuration changes. If Relational Migrator determines the 
+database needs configuration changes, it automatically generates a 
+SQL script with the required changes. It is recommended to have a 
+Database Administrator (DBA) review the commands in this script and 
+perform their execution on the database server. These instructions 
+configuration PostGreSQL for both types of sync jobs:
+
+.. include:: /includes/fact-short-sync-job-desc.rst
+
+About this Task
+---------------
+
+Relational Migrator uses the PostgreSQL ``pgoutput`` plugin for logical 
+decoding. This plug-in is included in PostgreSQL versions 10+, no 
+additional installation or configuration is required.
+
+Before you Begin
+----------------
+
+If PostgreSQL is configured as a cluster, Relational Migrator must 
+connect to the master server.
+
+Steps
+-----
+
+.. tabs::
+
+   .. tab:: Snapshot Jobs
+      :tabid: enable-snapshot-jobs
+
+      For snapshot jobs against PostGreSQL, the service account requires
+      schema ``USAGE`` and table ``SELECT`` permissions. Replace 
+      ``POSTGRES`` with the name of your service account user:
+
+      .. code-block:: sql
+         :copyable: true
+
+         GRANT USAGE ON SCHEMA SCHEMA_NAME TO POSTGRES;
+         GRANT SELECT ON TABLE SCHEMA_NAME.TABLE_NAME TO POSTGRES;
+
+   .. tab:: Continuous Jobs
+      :tabid: enable-continuous-jobs
+
+      .. procedure::
+         :style: normal
+
+         For continuous jobs against PostGreSQL, you must edit the 
+         configuration file, grant role permissions to the service 
+         account, and create a publication.
+
+         .. step:: Edit the configuration file 
+
+            Logical replication may not be enabled by default. To enable logical 
+            replication, change the `wal_level <https://postgresqlco.nf/doc/en/param/wal_level>`__ 
+            configuration in the 
+            `postgresql.conf configuration file <https://www.postgresql.org/docs/current/config-setting.html>`__. 
+            You *must restart the database instance* after changing the configuration file. 
+
```

Comment:
Can we add a quick note on how to do this in AWS RDS? The config file isn't exposed so you need to edit the RDS instance parameters. See 
https://dba.stackexchange.com/a/243576

Reply:
I added a link and some context to how to do this in AWS RDS to this step.